Shared/Models/Parse Subclasses/User/User+INPerson.swift

```diff
@@ -8,23 +8,34 @@
 
 import Foundation
 import Intents
+import UIKit
 
 extension User {
 
-    var inPerson: INPerson? {
+    func getINPerson() async -> INPerson? {
+
+        var inImage: INImage? = nil
+        if let data = try? await self.smallImage?.retrieveDataInBackground() {
+            if let image = UIImage(data: data),
+               let cgImage = image.cgImage {
+
+                let orientedImage = UIImage.init(cgImage: cgImage,
```

Comment:
I have no idea if this will fix the issue with image being the wrong orientation.

Reply:
Hi